src/features/run_snippets.ts

```diff
@@ -124,6 +124,11 @@ const isOnWordBoundary = (state: EditorState, triggerPos: number, to: number, wo
 }
 
 const trimWhitespace = (replacement: string, ctx: Context) => {
+	// Modify matrix expansion for inline math
+	const regex = new RegExp("\\n\\$0\\n", "g");
+  	replacement = replacement.replace(regex, " $0 "); 
+
+	// Trim whitespace
```

Comment:
This shouldn't be needed with the current default snippets (https://github.com/artisticat1/obsidian-latex-suite/blob/4021ed5c0305acb60808e5a40f9b4602f1142d0d/src/default_snippets.js#L239-L244).

We can add the ability to remove newlines when expanding snippets inside inline math, but it should probably be separate from the `trimWhitespace` function